keyboards/planck/keymaps/tylerwince/keymap.c

```diff
@@ -0,0 +1,251 @@
+#include QMK_KEYBOARD_H
+#include "muse.h"
+#include "eeprom.h"
+#include "keymap_german.h"
```

Comment:
I don't see where all these language-specific keycode mappings are used. Can they be removed?

Reply:
These are because the ZSA configurator adds them for intl support. 

However, the eeprom one isn't necessary.   
The rest can be removed.

Reply:
@drashna -- are you saying  i only need the `#include "muse.h"` line here? I am trying to get all the auto-generated code gone so I can manipulate the config easier